scripts/groot_387_add_states/script.py

```diff
@@ -0,0 +1,41 @@
+"""
+to run:
+django shell < scripts/groot_387_add_states/script.py
+"""
+
+from django.db.models import Count
+
+from cities_light.models import Region
+
+from brand.models.brand import Brand
+from brand.models.brand_state import StateLicensed
+from brand.models.state import State
+
+
+usa_state_data = Region.objects.filter(country_id=234)
+canada_state_data = Region.objects.filter(country_id=38)
+australia_state_data = Region.objects.filter(country_id=13)
+
+# Populate our State Model for USA, Canada, and Australia
+for [country_code, states] in [
+    ["US", usa_state_data],
+    ["CA", canada_state_data],
+    ["AU", australia_state_data],
+]:
+    for state in states:
+        State.objects.create(
```

Comment:
For scripts like these, it's really better to use `_, _ = get_or_create`. That way the tester can run them multiple times